.github/workflows/move_issue.yaml

```diff
@@ -4,19 +4,34 @@ on:
     types: [labeled]
 jobs: 
   gitIssueTransfer:
+    env:
+      bot_comment: "FYI: @DynamoDS/dynamo"
     strategy:
       matrix: 
         include: 
           - label: 'Revit'
             repoName: 'DynamoRevit'
           - label: 'Advance-Steel'
             repoName: 'Dynamo-Advance-Steel'
+          - label: 'Wishlist'
+            repoName: 'DynamoWishlist'
     name: Move issues to corresponding repository
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+
+      #Adds a comment to the issue before moving it
```

Comment:
I was thinking about after moving it, can you describe or give an example how would it looks like?

Reply:
![image](https://user-images.githubusercontent.com/66637541/102147907-4ae5ac80-3e4a-11eb-9001-a24d2134c73e.png)

This is a test I did on the repo I was testing the workflow, The creator of the issue will remain the same as in the original issue, and this will be added to the new issue thread. The message left by the bot is missing the "@" as this was a test but it is included for this pr. The author of the transfer will be the DynamoBot account created for that purpose.

Reply:
Since you mentioned this will be added to the new issue thread, should we update the comment saying `#Adds a comment to the issue after moving it`

Reply:
The comment is actually added before the issue is transferred and then everything gets moved with the issue to the new repo

Reply:
@Astul-Betizagasti OK, got you